### PR TITLE
Break kubemark to experiment with the presubmit

### PR DESCRIPTION
**Pull request by shyamjvs**
cc @liggitt @jdumars @mtaufen @kubernetes/sig-testing-bugs

### CODE REVIEW

**Comment by shyamjvs** [comment]
Ref the discussion shooting from https://github.com/kubernetes/kubernetes/pull/51307

File: pkg/kubemark/hollow_kubelet.go [hunk 1]
```diff
@@ -67,7 +67,7 @@ func NewHollowKubelet(
 	volumePlugins := empty_dir.ProbeVolumePlugins()
 	volumePlugins = append(volumePlugins, secret.ProbeVolumePlugins()...)
 	d := &kubelet.Dependencies{
-		KubeClient:        client,
```
Comment (liggitt) [inline pkg/kubemark/hollow_kubelet.go]: if this fails as expected, you might try simulating the issue the other PR caused (the "" dockershim)

Comment (shyamjvs) [reply pkg/kubemark/hollow_kubelet.go]: Yup... I was planning to do that. Before that wanted to check if it's something obvious with testinfra

Comment (shyamjvs) [reply pkg/kubemark/hollow_kubelet.go]: So it failed as expected. Trying now with the revert reverted.

**Comment by luxas** [comment]
@shyamjvs please don't create new branches on kubernetes/kubernetes, use your own fork instead...

**Comment by shyamjvs** [comment]
Sorry about it... I edited the file using github (my first time doing that) and it created the branch here. I should've missed something.
Will close it soon- just testing out stuff.

**Comment by shyamjvs** [comment]
Closing this branch as we've got the result. I've explained what happened here - https://github.com/kubernetes/kubernetes/issues/51856#issuecomment-326769410

**Pull request closed without merging**